stepactions/get-unreleased-bundle/0.1/README.MD

```diff
@@ -0,0 +1,27 @@
+# get-unreleased-bundle stepaction
+
+This StepAction retrieves the highest bundle version from a specified package and channel in the provided FBC fragment. 
+If package name and/or channel name are not specified, the step will determine the default package and/or channel name.
+
+Default package name determination:
+    
+    * If there is only one 'olm.package', it's name is returned
+    
+    * If there are multiple 'olm.package' entries, the package with the highest bundle version in its 'defaultChannel' is selected
+
+Default channel name determination:
+    
+    * The default channel name corresponds to the 'defaultChannel' entry of the selected package
+The StepAction checks whether the bundle is unreleased, specifically, not present in the Red Hat production Index Image (registry.redhat.io/redhat/redhat-operator-index). 
+If the bundle is unreleased, the StepAction returns it.
+## Parameters
+|name|description|default value|required|
+|---|---|---|---|
+|fbcFragment|A FBC fragment image.||true|
+|packageName|An OLM package name present in the fragment or leave it empty so the step will determine the default package name as described above.|If there is only one 'olm.package', it's name is returned. If there are multiple 'olm.package' entries, the package with the highest bundle version in its 'defaultChannel' is selected|false|
+|channelName|An OLM channel name or leave it empty so the step will determine the default channel name as described above.|The defaultChannel name for a given package.|false|
```

Comment:
I think that `packageName` and `channelName` params should have the [same value of `default value` as defined in the StepAction](https://github.com/konflux-ci/tekton-integration-catalog/pull/101/files#diff-adf7c1beb0e966cbd7dca54175fb246a9ba7091d839bda996b0c4be93253c175R24), i.e. empty string: `""`